demo/src/TableTab.jsx

```diff
@@ -29,16 +29,17 @@ const evenThenOddOrderingKey = (n) => {
     if (n <= 0 && remainder < 1) {
         // first negative even and zero ]...-3,-2,-1]
         return n / 2 - 1;
-    } else if (n > 0 && remainder < 1) {
+    }
+    if (n > 0 && remainder < 1) {
```

Comment:
I don't understand this change...

Reply:
https://eslint.org/docs/latest/rules/no-else-return